docs/_articles/en/apex/testing.md

```diff
@@ -3,27 +3,43 @@ title: Apex Tests
 lang: en
 ---
 
-You can run Apex tests from within a file or from the Apex Tests sidebar. The sidebar provides other useful features for working with your tests.
+Apex extension allows you to run Apex tests to verify the functionality of your code. You can retrieve code coverage results for your Apex classes and triggers every time you run one or more tests. To do this, [edit your workspace settings](https://code.visualstudio.com/docs/getstarted/settings) and set `salesforcedx-vscode-core.retrieve-test-code-coverage` to **true**.
 
 ## Explore Your Apex Tests
 
-The Apex Tests sidebar provides several features. Here, you can see all your Apex tests at a glance. You can run one test method, the test methods in one class, or all your tests. You can view the results of your last test run. And you can jump from those results to the corresponding lines in your code. To access this sidebar, click the beaker icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an `sfdx-project.json` file in its root directory.)
+The Test view in the Side Bar provides several features such as LWC Tests and Apex Tests. To access the Test view, click the beaker icon ({% octicon beaker %}) in the Activity Bar on the left side of the VS Code editor. If you don’t see this icon, make sure that the project contains an `sfdx-project.json` file in its root directory. If the Test view is empty, check if the [Java setup](../getting-started/java-setup) is configured correctly.
```

Comment:
I really like your use of ({% octicon beaker %}). Just as a heads up though - I'm not able to see it in the preview of the doc.